### PR TITLE
PerfScale 416 jobs

### PR DESCRIPTION
**Pull request by paigerube14**
Adding start to 4.16 self managed jobs and missing slack reporting

### CODE REVIEW

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-3nodes

**Comment by paigerube14** [comment]
/assign @jtaleric @svetsa-rh 
Thoughts on going ahead and adding a start for 4.16 runs on self-managed

**Comment by paigerube14** [comment]
The 3 node control plane test worked here: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/47151/rehearse-47151-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-3nodes/1742279614085992448

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      SIZE_VARIANT: large
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-aws
+  timeout: 6h0m0s
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml:38]: only nit, we are bumping the 120 node tests for control-plane to 7 hours.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m5.4xlarge
+      HUBBLE: "true"
+      KUBE_BURNER_VERSION: 1.8.0
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-aws-cilium
+  timeout: 6h0m0s
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml:273]: Same here to just be consistent.

**Review by jtaleric — COMMENTED** [review]
one nit wrt timeout

**Pull request merged**